extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx

```diff
@@ -357,7 +357,9 @@ export const TransactionDetails = ({ goBack }: { goBack: () => void }) => {
           url,
           networkDetails,
         );
+        return;
```

Comment:
does this do something the `return` on ln 388 does not?

Reply:
well I think we could safely drop it but in the case that we `scanTx` we don't need to manually `setLoading(false)` because it happens in the hook call. So the bug was that we relied on `isLoading` but didn't always set it correctly in the case here.
Dropping it should only result in calling that state setter once unnecessarily which is probably fine.